Sources/Sentry/SentryCrashAdapter.m

```diff
@@ -30,6 +31,16 @@ - (BOOL)isBeingTraced
     return sentrycrashdebug_isBeingTraced();
 }
 
+- (void)installAsyncHooks
+{
+    sentrycrash_install_async_hooks();
+}
+
+- (void)deactivateAsyncHooks
```

Comment:
l: why dont we call `uninstallAsyncHooks`? so it matches `installAsyncHooks`

Reply:
I kept the existing naming. I guess the name is intentionally `deactivate`, see
https://github.com/getsentry/sentry-cocoa/blob/65f4ece762dc954569a7fe71e3249bbd5747e5c6/Sources/SentryCrash/Recording/Tools/SentryHook.c#L298-L300

Reply:
makes sense then